pygsheets/cell.py

```diff
@@ -278,7 +296,7 @@ def get_json(self):
             nformat, pattern = self.format, ""
         return {"userEnteredFormat": {
                         "numberFormat": {
-                            "type": nformat.value,
```

Comment:
Yeah `format` is a tuple, its first item is the enum [see here](https://github.com/nithinmurali/pygsheets/blob/master/pygsheets/cell.py#L43), which we are getting  into `nformat`.